src/App.css

```diff
@@ -18,5 +18,38 @@ BEM conventions we are using:
 */
 
 .container {
+  max-width: 500px;
+  margin: 0 auto;
+  font-size: 20px;
+  padding: 40px;
+  position: relative;
+}
+
+.container__header {
+  margin-bottom: 40px;
+}
+
+.container__separator {
   text-align: center;
+  margin: 40px 0;
+}
+
+.text-field {
+  display: block;
+  box-sizing: border-box;
```

Comment:
We've put border-box box-sizing on this particular class, but do we want to set it globally for all elements?

Reply:
I'd worry that might be a bit counterintuitive/confusing, since a person wouldn't necessarily expect that behaviour for all elements. I just did this on a few form elements to easily make them 100% width without having to account for padding, border, etc.

Reply:
I think it makes sense for now to leave it at 100%. Looking ahead and the user stories coming up I think this may change. However, I think it is ok for now.

Reply:
Yeah, these are just barebones styles to make the app less painful to look at while building, hence trying to do the minimum. I expect we'll revisit and better optimize the CSS once we finalize a design.